script.js

```diff
@@ -0,0 +1,32 @@
+addField = function(){
+
+    $('.add').on('click', function(){
+        const parent = $(this).parent().attr('class')
+        const contentToAppend = parent + 'Inner'
+        const placeToAppend = parent + 'Content'
+        const addedContent = $(`.${contentToAppend}`).html()
+        $(`.${placeToAppend}`).append(addedContent)
+    })
+}
+
+const resumeBuilder = {
+}
+
+resumeBuilder.userInput = function(){
+    $('button').on('click', function(event){
+        event.preventDefault();
```

Comment:
Woo hoo. You found preventDefault. Took me like 10 minutes to remember how to use it.